api/all/build.gradle

```diff
@@ -11,10 +11,7 @@ ext.moduleName = "io.opentelemetry.api"
 archivesBaseName = "opentelemetry-api"
 
 dependencies {
-    api project(':api:context'),
-            project(':api:baggage'),
-            project(':api:common'),
-            project(':api:trace')
+    api project(':api:context')
```

Comment:
ok... just thinking here...  if `:api` is going to contain baggage and traces, why not context?